pkg/apis/network/v1alpha1/common.go

```diff
@@ -1,8 +1,8 @@
 package v1alpha1
 
 import (
-	"kubesphere.io/kubesphere/pkg/apis/network/v1alpha1/numorstring"
 	corev1 "k8s.io/api/core/v1"
+	"kubesphere.io/kubesphere/pkg/apis/network/v1alpha1/numorstring"
```

Comment:
import path is ambiguous , why import network package into devops? consider move `numorstring` to a more public package, such as `utils`

Reply:
just gofmt here